modules/examiner/test/ExaminerTest.php

```diff
@@ -29,36 +29,182 @@ class ExaminerTest extends LorisIntegrationTest
+
+        // Check the filter button
+        $filterButton = $this->webDriver->findElement(
+            WebDriverBy::Name("filter")
+        );
+        $this->assertEquals("submit", $filterButton->getAttribute("type"));
+
+        // Check the reset button
+        $resetButton = $this->webDriver->findElement(
+            WebDriverBy::Name("reset")
+        );
+        $this->assertEquals("button", $resetButton->getAttribute("type"));
+
+        $this->resetPermissions();
+    }
+
+    /**
+     * Tests that the Add Examiner form loads if the user has the correct permission
+     *
+     * @return void
+     */
+    function testAddExaminerFormLoadsWithPermission()
+    {
+        $this->setupPermissions(array("examiner_view"));
+        $this->safeGet($this->url . "/examiner/");
+
+        // Test that the selection filter appears
+        $bodyText = $this->webDriver->findElement(
+            WebDriverBy::id("lorisworkspace")
+        )->getText();
+        $this->assertContains("Add Examiner", $bodyText);
+
+        // Check the name input
+        $nameInput = $this->webDriver->findElement(
+            WebDriverBy::Name("addName")
+        );
+        $this->assertEquals("input", $nameInput->getTagName());
+
+        // Check the radiologist input
+        $radiologistInput = $this->webDriver->findElement(
+            WebDriverBy::Name("addRadiologist")
+        );
+        $this->assertEquals("input", $radiologistInput->getTagName());
+
+        // Check the site input
+        $siteInput = $this->webDriver->findElement(
+            WebDriverBy::Name("addSite")
         );
-        $bodyText = $this->webDriver
-            ->findElement(WebDriverBy::cssSelector("body"))->getText();
-        $this->assertContains("Edit Examiner", $bodyText);
-         */
+        $this->assertEquals("select", $siteInput->getTagName());
+
+        $this->resetPermissions();
+    }
+
+    /**
+     * Tests that the examiner table loads if the user has the correct permission
+     *
+     * @return void
+     */
+    function testResultTableLoadsWithPermission()
+    {
+        $this->setupPermissions(array("examiner_view"));
+        $this->safeGet($this->url . "/examiner/");
+
+        // Check the table column headers
+        $tableText = $this->webDriver->findElement(
+            WebDriverBy::cssSelector(".table-responsive")
+        )->getText();
+        $this->assertContains("No.", $tableText);
+        $this->assertContains("Examiner", $tableText);
+        $this->assertContains("Site", $tableText);
+        $this->assertContains("Radiologist", $tableText);
+
+        $this->resetPermissions();
+    }
+
+    /**
+     * Tests that the certification column loads if EnableCertification is set in
+     * the config
+     *
+     * @return void
+     */
+    /*function testExaminerLoadsCertificationElements()
+    {
+        $this->setupConfigSetting('EnableCertification', '1');
```

Comment:
Why are these commented out? If It's to skip them just mark the test as skipped, otherwise if it's unused delete it..